8.2.28 Release # DO NOT DELETE

su-sws-jira opened the pull request.

kaseybon committed:
STN-646: Update Utility Docs (#792)

* chore: improve humsci_basic CSS documentation

STN-646

* chore: moved admin specific styles to an admin directory

* chore: add missing utility classes from documentation

* refactor: remove duplicate style rule

* chore: document the dark inversion classes

* chore: compiled CSS

* docs: document which classes work with views

su-sws-jira committed:
8.2.28

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

pookmish committed:
fixed migrated content banner logic

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

Commit:
Feat(STN-597): update figcaption to new caption/credit styling in WYSIWYG (#801)

* feat(STN-597): update figcaption to new caption/credit styling

This updates the old figcaption styling to match our new universal caption/credit styling as a gradient overlay.

* chore: add ol to list styles for captions

* chore: compile css

Commit:
feat (STN- 645): Convert imported D7 classes (#803)

* feat(STN-645): Imported D7 content css class coverage

Aligning D7 and decanter styles in with our HS and HB utility classes so they have proper styling when they are imported in.

kaseybon committed:
fix: hb-raised-cards--uniform-height works with full width well class (#805)

STN-649

Commit:
STN-607: Implementation of Testimonial Paragraph/UI Pattern for views (#787)

* feat(STN-607): Implementation of Testimonial Paragraph/UI Pattern for views

* feat: create mixin for quote styles that can be used on both the blockquote and the testimonial

* feat: refactor testimonial pattern and continue styling

* feat: add testimonial configuration files

* feat: add styles for hb-raised-cards utility class

* feat: remove hb-raised-cards styles

Raised card styles will be addressed in a testimonial refinement card in a future PR

* feat: update behat test for the inclusion of the new testimonial paragraph type

* feat: add missing updates to behat test

* feat: address hb-dark-pattern styling issues

* feat: resolve issue with spacing when the testimonial is the first paragraph type on the page when the hb-dark-pattern utility class is applied in layout builder

* feat: refine testimonial styles for wells when used with the hb-dark-pattern class

* feat: fix spacing issues when the hb-dark-pattern utility class with rows and wells

* feat: add missing styles for color of dark inversion quotes

* feat: add compiled CSS

* feat: add new line to testimonial twig file to adhere to linting conventions

su-sws-jira committed:
Updated dependencies Feb 10 2021

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

kaseybon committed:
fix: only apply uniform height styles to comonents in a grid or row (#806)

Commit:
chore: compile css

joegl committed:
HSD8-988: Added ccsre to multi-site.. (#807)

Co-authored-by: joegl <joegl@users.noreply.github.com>

pookmish committed:
Sparkbox Sprint 33

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

pookmish committed:
Dont hotlink stage file proxy files

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

pookmish committed:
HSD8-993 Account for empty strings in the department

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

pookmish committed:
HSD8-1006 Use the default field value for images on profiles

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

pookmish committed:
HSD8-1003 Exclude testimonial paragraph type on older themes

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

pookmish committed:
HSD8-1004 Fix special characters in vertical linked cards

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

Commit:
STN-689 : HOTFIX Pull Decanter Mappings (#810)

* fix (STN-689): pull decanter mappings from recent work to convert imported D7 classes

* feat: add compiled CSS

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

Commit:
STN-682:  HOTFIX Carousel caption credit width change (#811)

* hotfix(STN-682): colorful caption credit width

* chore: compile css

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

pookmish committed:
Updated default config and content (#813)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

pookmish committed:
Check fo ids to update database

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs

su-sws-jira committed:
Updated dependencies Feb 17 2021

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr802-2gcugr3ko9ek1dicevm1zd8u1qnkskxj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601af38198552d211a559aae

Visual Diffs:
- https://dashboard.tugboat.qa/601af38198552d211a559aae/visualdiffs
